pylibs/unittests/test_basic.py

```diff
@@ -180,6 +180,38 @@ def assert_increasing(c0: Dict[str, int], c1: Dict[str, int]):
         c20 = ns.counters()
         assert_increasing(c10, c20)
 
+    def testConfigVisualization(self):
+        ns = self.ns
+        vopts = ns.config_visualization()
+        print('vopts', vopts)
+        for opt in ('broadcast_message', 'unicast_message', 'ack_message', 'router_table', 'child_table'):
+            self.assertTrue(opt in vopts)
+
+        vopts = ns.config_visualization(broadcast_message=False)
+        self.assertFalse(vopts['broadcast_message'])
+
+        vopts = ns.config_visualization(unicast_message=False)
+        self.assertFalse(vopts['broadcast_message'])
```

Comment:
Is testing `broadcast_message` after setting `unicast_message` intentional?

Reply:
Actually yes. The `self.assertFalse(vopts['broadcast_message'])` statement is here to check if the configuration of `broadcast_message` is kept unchanged in `ns.config_visualization(unicast_message=False)`.

Reply:
Why not check other configurations?

Reply:
yes, I will do that

Reply:
revised the test